lib/HTML/Restrict.pm

```diff
@@ -71,6 +71,12 @@ has trim => (
     default => 1,
 );
 
+has process_text => (
```

Comment:
This was my first thought as well, but I think we can tweak this slightly to make the API even more flexible. How about we follow the pattern of `replace_img`?

```perl
has replace_img => (
    is      => 'rw',
    isa     => Bool | CodeRef,
    default => 0,
);
```

I think the name of the attribute you've chosen makes sense.

If we follow this pattern we can either process the text as is currently the default, do nothing or allow the user to provide their own processing sub. That feels like a pretty flexible solution.

Reply:
Sounds good, 
Ill get that in ASAP